impacket/krb5/kerberosv5.py

```diff
@@ -712,7 +712,7 @@ def getErrorPacket( self ):
         return self.packet
 
     def getErrorString( self ):
-        return str(self)
+        return nt_errors.ERROR_MESSAGES[self.error]
```

Comment:
Dictionary to handle error messages in kerberosv5 should be `krb5.constants`

```suggestion
        return constants.ERROR_MESSAGES[self.error]
```